projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas-header.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'go-off-canvas-header-actions',
```

Comment:
The class name for this component is `GoOffCanvasHeaderComponent`, but the selector differs. Should we standardize those? I think I would lean towards, `go-off-canvas-header`

Reply:
good catch. I went back and forth with the name of the component and forgot to update the selector.